myriad-scheduler/src/main/java/com/ebay/myriad/configuration/ServiceConfiguration.java

```diff
@@ -129,4 +142,96 @@ public String getServiceOpts() {
     return serviceOptsName;
   }
 
+  /**
+   * Custom Serializer
+   * @param <T>
+   */
+  public static class OptionalSerializer<T> extends JsonSerializer<Optional<T>> {
```

Comment:
Please use a new file to put this class in, as it is used by multiple configuration classes.